src/lib/paths.ts

```diff
@@ -10,7 +10,10 @@ export const pathForStudyJobCode = (parts: MinimalJobInfo) => `${pathForStudyJob
 export const pathForStudyJobCodeFile = (parts: MinimalJobInfo, fileName: string) =>
     `${pathForStudyJobCode(parts)}/${sanitizeFileName(fileName)}`
 
-export const pathForStudyDocuments = (parts: MinimalStudyInfo, docType: StudyDocumentType, fileName: string) =>
+export const pathForStudyDocuments = (parts: MinimalStudyInfo, docType: StudyDocumentType) =>
+    `${pathForStudy(parts)}/docs/${docType}`
+
+export const pathForStudyDocumentFile = (parts: MinimalStudyInfo, docType: StudyDocumentType, fileName: string) =>
     `${pathForStudy(parts)}/docs/${docType}/${sanitizeFileName(fileName)}`
```

Comment:
the `pathForStudyDocumentFile` can call `pathForStudyDocuments` and append the `${sanitizeFileName(fileName)` to it's return value